foxglove/cmd/attachments.go

```diff
@@ -0,0 +1,75 @@
+package cmd
+
+import (
+	"fmt"
+	"io"
+	"os"
+
+	"github.com/foxglove/foxglove-cli/foxglove/console"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+func newListAttachmentsCommand(params *baseParams) *cobra.Command {
+	var format string
+	var importID string
+	var recordingID string
+	attachmentsListCmd := &cobra.Command{
+		Use:   "list",
+		Short: "List MCAP attachments",
+		Run: func(cmd *cobra.Command, args []string) {
+			client := console.NewRemoteFoxgloveClient(
+				*params.baseURL, *params.clientID,
+				viper.GetString("bearer_token"),
+				params.userAgent,
+			)
+			err := renderList(
+				os.Stdout,
+				&console.AttachmentsRequest{
+					ImportID:    importID,
+					RecordingID: recordingID,
+				},
+				client.Attachments,
+				format,
+			)
+			if err != nil {
+				fmt.Fprintf(os.Stderr, "Failed to list imports: %s\n", err)
+				os.Exit(1)
+			}
+		},
+	}
+	attachmentsListCmd.InheritedFlags()
+	attachmentsListCmd.PersistentFlags().StringVarP(&importID, "import-id", "", "", "Import ID")
+	attachmentsListCmd.PersistentFlags().StringVarP(&recordingID, "recording-id", "", "", "Recording ID")
+	AddFormatFlag(attachmentsListCmd, &format)
+	return attachmentsListCmd
+}
+
+func newGetAttachmentCmd(params *baseParams) *cobra.Command {
+	attachmentsGetCmd := &cobra.Command{
+		Use:   "get",
+		Short: "Get an MCAP attachment by ID",
+		Args:  cobra.ExactArgs(1),
+		Run: func(cmd *cobra.Command, args []string) {
+			attachmentID := args[0]
+			client := console.NewRemoteFoxgloveClient(
+				*params.baseURL, *params.clientID,
+				viper.GetString("bearer_token"),
+				params.userAgent,
+			)
+			rc, err := client.Attachment(attachmentID)
+			if err != nil {
+				fmt.Fprintf(os.Stderr, "Failed to fetch attachment: %s\n", err)
+				os.Exit(1)
+			}
+			defer rc.Close()
+			_, err = io.Copy(os.Stdout, rc)
```

Comment:
I'm on the fence:
* for text output, writing straight to stdout is pretty nice
* for non-text output and text output, redirecting to a file is pretty nice. If I'm redirecting I don't care what the content disposition says.
* If we want to use the content disposition, the I think the behavior would probably be something like "foxglove attachments get xyz" and then it would write to the location the header specifies, and log some message telling you where to find the data?

I kind of feel like 2 is more convenient for a CLI user than 3. 3 also introduces complexity around what should happen if a file with that name exists already. Easy to resolve, but I think it's a little less direct than the experience I would expect from a CLI.